lsb_vsx/lsb_vsx.py

```diff
@@ -15,27 +15,28 @@ def get_subtests_numbers(name: str) -> Iterator[int]:
     # Example of selected tests using curly braces: {1,3-6,8-12}
     selected_tests = match.group(1)
 
-    nums = set()
+    nums = []
```

Comment:
Why not set? The test case numbers shouldn't be duplicated

Reply:
The order of items in a set isn’t preserved since sets are hash-based.

The numbers shouldn’t be duplicated because they’re manually defined in `test.yaml`.